morphia/src/test/java/org/mongodb/morphia/entities/EmbeddedType.java

```diff
@@ -16,6 +16,9 @@
 
 package org.mongodb.morphia.entities;
 
+import org.mongodb.morphia.annotations.Embedded;
```

Comment:
Validation in of the tests fails when this class is mapped without it.  The use of this test entity has been expanded slightly and now errors without it.